deployments/goerli/usdc/roots.json

```diff
@@ -8,5 +8,6 @@
     "comet": "0x44Ac0af6F39c98d59Ed6D16404210656dB912Ffd",
     "configurator": "0xdE0e9466F406Eb0922746861Bc09785bED912C80",
     "rewards": "0xef9e070044d62C38D2e316146dDe92AD02CF2c2c",
-    "bulker": "0x2677D9ca7477A81A394C324B97F3D7907C6727Fb"
+    "bulker": "0x2677D9ca7477A81A394C324B97F3D7907C6727Fb",
+    "fxRoot": "0x3d1d3e34f7fb6d26245e6640e1c50710efff15ba"
```

Comment:
Sorry to go back and forth on this, but I guess it could be a bit weird to put `fxRoot` in roots, since currently roots are specific to a certain deployment. What we really want is for `fxRoot` to be in roots for all of goerli, not just the usdc deployment of goerli. So maybe it's better to just hardcode this address in the migration script until we have the concept of a network-level roots?

Reply:
As a counter-argument: we currently specify a specific parent _deployment_ (not just a parent network).

So when we're attaching an `auxiliaryBase` to a scenario base, we're saying that governance of this bridged deployment (e.g. Mumbai/USDC) is the governor of a specific base (e.g. Goerli/USDC).

This makes sense, since it would be possible (albeit unlikely) for there to be a second deployment on the parent network (e.g. Goerli/DAI) that was governed by an entirely different GovernorBravo/Timelock/etc.

And while we don't have shared roots for a network (so `fxRoot` would be duplicated in `deployments/goerli/dai/roots.json`), we do have the ability to share relations (the way we have Goerli's relations inherit from base relations), so there would a limit to how much code would be duplicated across deployments.

Reply:
Fair enough, that makes a lot of sense!

Reply:
synced offline.

- [ ] better approach seems like it would be to add `fxRoot` to the Goerli deploy script, so that this `fxRoot` address gets added automatically to `deployment/goerli/usdc/roots.json`

Reply:
removed hard-coded `fxRoot` from Goerli roots; using `.existing` to pull it in as part of the Goerli deploy script